.circleci/config.yml

```diff
@@ -147,6 +147,13 @@ commands:
           command: |
             sudo apt-get update -y && sudo apt-get install -y libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev
 
+  install-streaming-compress-libs:
```

Comment:
n00b: 
1. Does WAL compression only work over streaming compress libs?
2. Do we have lz4 streaming compress lib on the circleci box? If so, can WAL compression us the lz4 one?

Reply:
According to https://github.com/facebook/rocksdb/blob/main/util/compression.h#L526, ZSTD so far is the only lib that may support streaming compression.

Reply:
> Does WAL compression only work over streaming compress libs?

Yes